pkg/common/util.go

```diff
@@ -42,6 +42,8 @@ func ParseAndEscapeArgs(args []string) string {
 		if runtime.GOOS == "windows" {
 			quotedArg := "\"" + strings.Replace(strings.Trim(arg, "\""), "\"", "`\"", -1) + "\""
 			commandWithArguments.WriteString(quotedArg + " ")
+		} else if runtime.GOOS == "darwin" {
+			commandWithArguments.WriteString(arg + " ")
```

Comment:
Are you sure that the arguments don't have to be escaped? I thought that it would behave mostly the same since mac is using a unix shell aswell?

Did you encounter any errors / issues when using the same escaping logic thats used for linux? (default case in that function)

Reply:
@PhilippHeuer just double checked and confirm that there is no need in custom quoting rules for `darwin`. I had some issues with quotes, but looks like problem was not in this part of code. Good catch! Thanks!